Video Player/app.js

```diff
@@ -0,0 +1,56 @@
+const player=document.querySelector(".player");
+const video=player.querySelector(".viewer");
+const progress=player.querySelector(".progress");
+const progressBar=player.querySelector(".progress__filled");
+const toggle = player.querySelector(".toggle");
+const skipButtons=player.querySelectorAll("[data-skip]");
+const ranges=player.querySelector("range");
+//player-slider: video kaydırıcı
+
+//lets write the functions
+function togglePlay(){
+    //.paused -> video is play or pause check
+    //if video paused then set ►, or ❚ ❚
+    if(video.paused){
+        video.play();
+    }
+    else{
+        video.pause();
+    } 
+}
+
+function updateButton(){
+    const icon= this.paused ? '►' : '❚ ❚';
+    console.log(icon);
+    toggle.textContent=icon;
```

Comment:
Burada icon tegiskenine gerek yok. 
`    toggle.textContent = this.paused ? '►' : '❚ ❚';`